.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+    "python.testing.pytestArgs": [
+        "test_unstructured"
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true,
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "main"
+    ]
+}
```

Comment:
this could be proposed in a separate PR, perhaps?

Reply:
ohh no, also, accidental